optuna/integration/lightgbm_tuner/optimize.py

```diff
@@ -382,6 +361,7 @@ def __init__(
         )  # type: Dict[str, Any]
         self._parse_args(*args, **kwargs)
         self._best_booster_with_trial_number = None  # type: Optional[Tuple[lgb.Booster, int]]
+        self.start_time = None  # type: Optional[float]
```

Comment:
[IMO] how about making this attribute private?

Reply:
That makes sense. I'll fix it.

Reply:
Thanks! Resolved by 3e09fc4.